native/src/components/CollapsibleItem.tsx

```diff
@@ -67,7 +67,7 @@ const CollapsibleItem = ({
       <Collapse
         isExpanded={isExpanded}
         onToggle={() => setIsExpanded(!isExpanded)}
-        touchableOpacityProps={{ activeOpacity: 1 }}>
+        PressableProps={{ activeOpacity: 1 }}>
```

Comment:
Where does this come from?

Reply:
Good catch, just did search/replace all.